x-pack/plugins/observability_solution/apm/server/lib/helpers/get_apm_alerts_client.ts

```diff
@@ -31,7 +32,7 @@ export async function getApmAlertsClient({
 
   const ruleRegistryPluginStart = await plugins.ruleRegistry.start();
   const alertsClient = await ruleRegistryPluginStart.getRacClientWithRequest(request);
-  const apmAlertsIndices = await alertsClient.getAuthorizedAlertsIndices(['apm']);
+  const apmAlertsIndices = await alertsClient.getAuthorizedAlertsIndices(apmAlertingFeatureIds);
```

Comment:
Should we update it also [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/infra/server/lib/helpers/get_infra_alerts_client.ts#L29) for consistency (Used in the host view)